src/NzbDrone.Core/Notifications/Webhook/Webhook.cs

```diff
@@ -0,0 +1,62 @@
+﻿
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Core.Tv;
+using NzbDrone.Common.Extensions;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    public class Webhook : NotificationBase<WebhookSettings>
+    {
+        private readonly IWebhookService _service;
+
+        public Webhook(IWebhookService service)
+        {
+            _service = service;
+        }
+
+        public override string Link
+        {
+            get { return "https://github.com/Sonarr/Sonarr/wiki/Webhook"; }
+        }
+
+        public override void OnGrab(string message)
+        {
+        }
+
+        public override void OnDownload(DownloadMessage message)
+        {
+            _service.OnDownload(message.Series, message.EpisodeFile, Settings);
+        }
+
+        public override void OnRename(Series series)
+        {
+            _service.OnRename(series, Settings);
+        }
+
+        public override bool SupportsOnGrab
+        {
+            get
+            {
+                return false;
```

Comment:
How about we leave this one disabled, and I create a new patch that turns onGrab from being a string to something like DownloadMessage object?

Least then the webhook could post more than just a string?

I'm in no way against it, it just seems like a kinda hack.
I guess I could throw in the kinda hack until the next patch is done.